instant-xml-macros/src/lib.rs

```diff
@@ -33,19 +33,22 @@ pub(crate) fn get_namespaces(
 
     if name == "namespace" {
         let mut iter = list.nested.iter();
-        if let Some(NestedMeta::Lit(Lit::Str(v))) = iter.next() {
+        let mut next = iter.next();
+        if let Some(NestedMeta::Lit(Lit::Str(v))) = next {
```

Comment:
What problem is this trying to solve? The iterator state is in `iter` so it should not be necessary to keep `next` as a separate binding here.

Reply:
It is about default namespace, to be able to parse something like:
`#[xml(namespace(bar = "BAZ", foo = "BAR"))]`
and 
`#[xml(namespace("BAZ", foo = "BAR"))]`
but not
`#[xml(namespace( foo = "BAR",  "BAZ"))]`
or
`#[xml(namespace("BAZ", "BAR")]`

Reply:
I was referring to the particular change of keeping a separate `next` binding instead of referring to `iter.next()` directly.